core/embed/rust/src/ui/model_mercury/layout.rs

```diff
@@ -329,6 +317,44 @@ extern "C" fn new_confirm_blob(n_args: usize, args: *const Obj, kwargs: *mut Map
     unsafe { util::try_with_args_and_kwargs(n_args, args, kwargs, block) }
 }
 
+extern "C" fn new_confirm_blob_intro(n_args: usize, args: *const Obj, kwargs: *mut Map) -> Obj {
```

Comment:
I don't love it as it creates `mercury` specific layout. But probably fine for now.

Reply:
concern linked in a new issue below